applications/machine_learning/sample.yaml

```diff
@@ -5,6 +5,7 @@ tests:
   applications.machine_learning.zdebug:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 thingy52_nrf52832 thingy53_nrf5340_cpuapp
+    platform_exclude: thingy53_nrf5340_cpuappns
```

Comment:
Hasn't "ns" suffix been replaced with "_ns"? The scripts seem to support both, but all docs now refer to "_ns" versions.

Reply:
It seems like we cannot execute CMake for `thingy53_nrf5340_cpuapp_ns` (it states that board is not supported). Maybe Thingy:53 still refers to old naming convention, but in that case we must use it too.

Reply:
This was changed by @mbolivar-nordic  if memory serves me right.

Reply:
Correct, boards has been renamed from `ns` --> `_ns`, for the  nRF5340dk in this commit:
https://github.com/nrfconnect/sdk-zephyr/commit/b8c9dc169e63dd37536d7381103a038fa79a69f2

But apparently Thingy53 was not part of that renaming, it still has the old `ns` naming scheme.